src/twisted/runner/test/test_procmon.py

```diff
@@ -279,6 +284,16 @@ def test_startProcessUnknownKeyError(self):
         """
         self.assertRaises(KeyError, self.pm.startProcess, "foo")
 
+    def test_startProcessSpawnException(self):
+        """
+        L{IReactorProcess.spawnProcess} might raise C{OSError}, ensure it
+        is caught and process is restarted
+        """
+        self.reactor._spawn_resource_unavailable = True
+        self.pm.addProcess("foo", ["foo"])
+        self.pm.startProcess("foo")
+        self.assertEquals(self.pm.delay["foo"], 4)
```

Comment:
This test is hard to read.

Maybe update the docstring of the test to explain what is going on here, or add an comment
```suggestion
        # The process is not started,
        # instead a restart is scheduled in 4 seconds,
        # where 4 seconds is the initial minumum interval.
        self.assertEquals(self.pm.delay["foo"], 4)
```

btw... it would be nice to have the value `4` part of the setup of this test, to make it easier to understand what that value 4 stand for.